card/lib/card/core_migration.rb

```diff
@@ -2,15 +2,15 @@
 
 class Card::CoreMigration < Card::Migration
   def migration_paths
-    Card::Migration.paths :core_cards
+    Cardio.migration_paths :core_cards
```

Comment:
Some migration related class/module methods were moved so Card isn't referenced when not yet loaded.